nginx.conf

```diff
@@ -1,6 +1,6 @@
 # Flask backend is listening on port 6000 on host called yaptide_flask
 upstream backend_flask {
-    server yaptide_flask:6000;
+    server yaptide-4.yaptide.svc.cluster.local:6000;
```

Comment:
This one should be done in much smarter way, as it will probably break the way app is deployed using docker compose.